.gitignore

```diff
@@ -8,5 +8,6 @@ webtau.report.html
 .vertx
 webtau-http-groovy/doc-artifacts/*
 webtau-feature-testing/doc-artifacts/*
+webtau-feature-testing/examples/doc-artifacts/*
```

Comment:
Unrelated to release.md changes obviously but I happen to have started a totally clean/new clone locally and spotted some untracked files appearing.